drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -460,6 +460,11 @@ static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len)
 
 #endif /* MODULE_NETDEV_IEEE802154_OQPSK */
 
+        case NETOPT_RANDOM:
+            assert(max_len >= sizeof(uint32_t));
+            at86rf2xx_get_random(dev, (uint8_t*)val, sizeof(val));
+            break;
+
```

Comment:
```suggestion
 #if AT86RF2XX_RANDOM_NUMBER_GENERATOR
        case NETOPT_RANDOM:
            assert(max_len >= sizeof(uint32_t));
            at86rf2xx_get_random(dev, (uint8_t*)val, sizeof(val));
            break;
#endif

```

Looks like not every revision of the transceiver has claims about the entropy for the RSSI value in the datasheet. E.g. see

https://github.com/RIOT-OS/RIOT/blob/932c58d89f732b86988219d5f17dd3ef6842524d/drivers/include/at86rf2xx.h#L124-L139